src/main/java/com/kmush12/CurrencyConverter/Converter/CurrencyConverterController.java

```diff
@@ -1,15 +1,13 @@
 package com.kmush12.CurrencyConverter.Converter;
 
-
 import com.kmush12.CurrencyConverter.Exchange.ExchangeRequest;
 import org.springframework.beans.factory.annotation.Autowired;
 import org.springframework.http.HttpStatus;
 import org.springframework.http.ResponseEntity;
 import org.springframework.web.bind.annotation.*;
 
-
 @RestController
-@RequestMapping("/convert")
+@RequestMapping("/currencyConverter")
```

Comment:
https://betterprogramming.pub/10-best-practices-for-naming-rest-api-endpoints-442ae592a3a0

https://www.freecodecamp.org/news/rest-api-best-practices-rest-endpoint-design-examples/

popatrz co w tym API jest głównym obiektem i zmień te nazwy endpointów na bardziej czytelne